extensions/3rdparty/LyricWiki/LyricFind/LyricFindHooks.class.php

```diff
@@ -54,7 +58,7 @@ static public function onAlternateEdit(EditPage $editPage) {
 					$blockEdit = true;
 				}
 			}
-		} else {
+		} elseif ( !$isCrawler ) {
```

Comment:
TBH I'm not sure if we should go with this check, it will allow bots to create pages that are prohibited by LyricFind (whatever that means)